src/test/kotlin/com/charleskorn/kaml/testobjects/TestObjects.kt

```diff
@@ -63,3 +79,126 @@ val sealedModule = SerializersModule {
         TestSealedStructure.SimpleSealedString::class with TestSealedStructure.SimpleSealedString.serializer()
     }
 }
+
```

Comment:
This file is getting quite large - how about we split the polymorphic serialization stuff out to its own file in this folder?